File: board-room/src/main/java/net/stzups/board/server/websocket/protocol/client/messages/ClientMessageGetInvite.java
package net.stzups.board.server.websocket.protocol.client.messages;

import io.netty.buffer.ByteBuf;
import net.stzups.board.server.websocket.protocol.client.ClientMessage;
import net.stzups.board.server.websocket.protocol.client.ClientMessageType;

public class ClientMessageGetInvite extends ClientMessage {
    public ClientMessageGetInvite(ByteBuf byteBuf) {
        super(ClientMessageType.GET_INVITE);
    }
}

File: board-frontend/html/scripts/protocol/client/messages/ClientMessageGetInvite.js
import ClientMessageType from "../ClientMessageType.js";
import ClientMessage from "../ClientMessage.js";

export default class ClientMessageGetInvite extends ClientMessage {
    constructor() {
        super(ClientMessageType.GET_INVITE);
    }

    getBufferSize() {
        return super.getBufferSize();
    }

    serialize(writer) {
        super.serialize(writer);
    }
}